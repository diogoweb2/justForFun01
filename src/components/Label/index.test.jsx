import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Label from "./index";

let wrapper;
describe("Label", () => {
  it("should render", () => {
    wrapper = shallow(<Label>test</Label>);
    expect(wrapper).toHaveLength(1);
  });

  it("should match snapshot...", () => {
    wrapper = shallow(<Label>test</Label>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
