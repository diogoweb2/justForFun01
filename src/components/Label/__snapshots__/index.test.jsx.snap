// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should match snapshot... 1`] = `
<label>
  test
</label>
`;
